server.sh

```diff
@@ -226,13 +236,13 @@ for driver in "${!drivers[@]}"; do
 	>&2 echo "Created loadbalancer ${lb_id}"
 	lb_ids+=("${lb_id}")
 
-	lb_listener_id="$(openstack loadbalancer listener create --wait --name "$name" -f value -c id --protocol TCP --protocol-port 22 "$lb_id")"
+	lb_listener_id="$(openstack loadbalancer listener create --wait --name "$name" -f value -c id --protocol TCP --protocol-port 80 "$lb_id")"
```

Comment:
since you changed the port, I guess we could remove the security group on [line 84](https://github.com/shiftstack/shiftstack-ci/pull/199/files#diff-14d1967cdbd3938fb362411b147b4e44ab53b6a45a2c11a6bdd0c0d3c1658e54R184)

but you can also leave it here for debug purpose, no strong opinion

Reply:
I did actually initially take this out along with the icmp one.. However I decided not to remove them because IIUC there are at least theoretically other users of this script?

I wonder if @pierreprinetti knows if these should stay or not.

Reply:
There are not afaik, and actually  I believe we could lower complexity by removing options not relevant for the liveness check. And rename it accordingly